internal/e2e/incompatible_by_default/incompatible_by_default_test.go

```diff
@@ -76,11 +70,3 @@ func TestRunWithPlatforms(t *testing.T) {
 
 	ibazel.ExpectOutput("hello!")
 }
-
-func TestRunWithConfig(t *testing.T) {
```

Comment:
Note to reviewing self. This is fine. We really don't need to test in *EVERY* test case that `--config` works properly. We should have a single test to validate that and all the other test cases should be explicit in their args